index.php

```diff
@@ -1,14 +1,15 @@
 <?php
     require __DIR__ . '/php/inc/header.tpl.php';
     require __DIR__ . '/php/inc/functions.php';
-    require __DIR__ . '/php/config.php';
+    require __DIR__ . '/php/classes/DB.php';
     require __DIR__ . '/php/inc/data.php';
     
     // Vérifiez si l'utilisateur est connecté, sinon redirigez-le vers la page de connexion
```

Comment:
You should avoid comments that are not really useful and just bloat the code if you want to keep your codebase readable and maintainable. 

Also, all your codebase, including comments, should be in english, just in case you want a non french speaker to work on it